packages/components/.storybook/tokens.stories.mdx

```diff
@@ -12,28 +12,41 @@ import styles from "./tokens.module.css";
 ## EDS Colors
 
 export const mapEDSColorToTokenArray = (color) => {
-  return Object.keys(tokens.eds.color[color])
+  // Handle nested color names
+  const colorKeys = color.split("-");
+  const colorTokens = colorKeys.reduce((obj, key) => {
+    return obj[key];
+  }, tokens.eds.color);
+  return Object.keys(colorTokens)
     .sort((shade1, shade2) => shade1.localeCompare(shade2))
     .map((shade) => ({
-      name: `$eds-color-${color}-${shade}`,
```

Comment:
Was the `$` just being added before because we previously wanted to document the scss variables, instead of the css variables?

Reply:
yep!